src/ui/components/SelectCouncilAccount.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import {View, FlatList, StyleSheet} from 'react-native';
+import {Menu, List, Caption, Icon, useTheme, Divider} from '@ui/library';
+import {Account as AccountType} from 'context/AccountsContext';
+import globalStyles from '@ui/styles';
+import Identicon from '@polkadot/reactnative-identicon';
+import {Account as SubstrateChainAccount} from 'src/api/hooks/useAccount';
+import {Account} from './Account/Account';
+import {useCouncilAccounts} from 'src/hooks/useCouncilAccounts';
+import {AccountItem} from './SelectAccount';
+
+type Props = {
+  onSelect: (account: SelectedAccount) => void;
+};
+
+type SelectedAccount = {
+  account: AccountType;
+  accountInfo?: SubstrateChainAccount;
+};
+
+export function SelectCouncilAccount({onSelect}: Props) {
```

Comment:
This component is doing the same as `SelectAccount`,  let's add a new prop `accounts` to `SelectAccount` and remove this one